charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```diff
@@ -45,6 +45,24 @@ spec:
           /sbin/sysctl -w net.ipv6.conf.all.disable_ipv6=1 || echo "IPv6 is already disabled. Doing nothing..."
         securityContext:
           privileged: true
+{{- end }}
+{{- if (semverCompare ">= 1.16" .Values.kubernetesVersion) }}
```

Comment:
Why is the init container only added for >= 1.16? Also, I've forgotten why switching the kube-proxy mode is prohibited only during 1.14.1-1.16 versions. What about earlier versions (< 1.14)? Does the switch work here? If yes, why is this init container not needed?

Reply:
kube proxy used autocleanup in the prior versions, see https://github.com/kubernetes/kubernetes/pull/76109

Reply:
Thank you!